NomadPlat/src/Linux/PlatformWindowLinux.cpp

```
@@ -0,0 +1,307 @@
+
+            case SDL_KEYDOWN:
+            case SDL_KEYUP:
+                if (m_keyCallback) {
+                    KeyCode key = translateKey(e.key.keysym.sym);
+                    KeyModifiers mods = getModifiers(e.key.keysym.mod);
+                    m_keyCallback(key, e.type == SDL_KEYDOWN, mods);
+                }
+                break;
+
+            case SDL_TEXTINPUT:
+                if (m_charCallback) {
```

Clang (14.0.6)

[note] +3, including nesting penalty of 2, nesting level increased to 3

(clang)